Header      header
time        timecode
uint16      sample_rate
uint16      chunk_size
uint16[]    shape
string      info
float32[]   data

